main/src/com/google/refine/ProjectMetadata.java

```diff
@@ -391,7 +391,7 @@ public void setAnyField(String metaName, String valueString) {
         } catch (NoSuchFieldException e) {
             updateUserMetadata(metaName, valueString);
         } catch (SecurityException | IllegalArgumentException | IllegalAccessException e) {
-            logger.error(ExceptionUtils.getFullStackTrace(e));
+            logger.error("Exception setting field " + metaName, e);
```

Comment:
```suggestion
            logger.error("Exception setting field " + metaName + "with value " + valueString, e);
```

Reply:
optional, but maybe more helpful, especially if we later change our Project metadata/value?